### PR TITLE
[2-dev] Update to Dart 2, and Angular 5-alpha for web-angular

### PR DESCRIPTION
**Pull request by chalin**
Note that I've created a 2-dev branch. This PR is set over 2-dev.

cc @kevmoo

### CODE REVIEW

File: templates/web-angular/pubspec.yaml [hunk 1]
```diff
@@ -5,29 +5,18 @@ version: 0.0.1
 #author: __author__ <email@example.com>
 
 environment:
-  sdk: '>=1.24.0 <2.0.0'
+  sdk: '>=2.0.0-dev.3.0 <2.0.0'
 
 dependencies:
-  angular: ^4.0.0
-  angular_components: ^0.8.0
+  angular: ^5.0.0-alpha
```
Comment (kevmoo) [inline templates/web-angular/pubspec.yaml]: careful here. I think the build bits were only activated in `^5.0.0-alpha+3`

Comment (kevmoo) [reply templates/web-angular/pubspec.yaml]: https://pub.dartlang.org/packages/angular/versions/5.0.0-alpha+3#-changelog-tab-

Comment (chalin) [reply templates/web-angular/pubspec.yaml]: Good catch! Fixed.

**Comment by chalin** [comment]
@kwalrath, @kevmoo - if either of you can approve this I'll merge it into the 2-dev branch.

File: .travis.yml [hunk 2]
```diff
@@ -11,28 +11,26 @@ env: # important: w/o this top-level env, no job will be allowed to fail.
...
+    # https://github.com/dart-lang/angular/issues/783, skip Angular tests because of pageloader issue
+    # - CI_JOB=tool/web-angular-test.sh
 
-matrix:
-  exclude:
-  - dart: dev
-    env: CI_JOB=tool/version-check-and-update.sh
-  - dart: dev
-    env: CI_JOB=tool/build-check.sh
+# matrix:
```
Comment (kevmoo) [inline .travis.yml]: Eh, If we're ditching stable for dev, let's just delete these lines, right?

File: .travis.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 language: dart
 sudo: false
 dart:
-  - stable
+  # - stable
```
Comment (kevmoo) [inline .travis.yml]: We're ditching stable – delete this line?

Or add a comment that we'll add it back once Dart v2 is stable?

Comment (chalin) [reply .travis.yml]: Deleted

**Comment by chalin** [comment]
@kevmoo - I've cleaned out the commented lines from the `.travis.yml` file.

File: .travis.yml (hunk 2 above)
Comment (chalin) [reply .travis.yml]: Deleted

**Pull request merged**